sts-mock/src/token/handlerDependencies.ts

```diff
@@ -0,0 +1,13 @@
+import { Logger as PowertoolsLogger } from "@aws-lambda-powertools/logger";
+import { MessageName, registeredLogs } from "./registeredLogs";
+import { Logger } from "../../../backend-api/src/functions/services/logging/logger";
```

Comment:
How do you feel about this? I wonder whether we should look at publishing our logger as an git npm package? Definitely not for something in the short-term

Reply:
strategically I'd like to explore publishing an npm module within Github. This is a change to our architecture patterns for mobile so needs a bit of thought/agreement from the wider pod. 

I think we could keep as is for now given this is a mock asset.